colin-api/src/colin_api/models/business.py

```diff
@@ -671,3 +688,66 @@ def reset_corp_states(cls, cursor, event_ids: List):
         except Exception as err:
             current_app.logger.error(f'Error in Business: Failed reset ended corp_state rows for events {event_ids}')
             raise err
+
+    @classmethod
+    def find_by_corp_num(cls, corp_num: str, con=None) -> Business:
+        """Return a Business by corp_num."""
+        business = None
+        try:
+            # get record
+            if not con:
+                con = DB.connection
+                # con.begin()
+
+            cursor = con.cursor()
+            cursor.execute(
+                """
+                select corp.corp_num, corp_typ_cd, can_jur_typ_cd, othr_juris_desc, home_juris_num
+                from CORPORATION corp
+                    left join JURISDICTION on JURISDICTION.corp_num = corp.corp_num
```

Comment:
Update this line to `on jurisdiction.corp_num = corp.corp_num and jurisdiction.end_event_id is null`

Reply:
I updated